src/app/TranscriptionFactor/[species]/[factor]/[detail]/MotifEnrichmentSELEX.tsx

```diff
@@ -51,6 +51,10 @@ import { downloadData, downloadSVGElementAsSVG } from "@/utilities/svgdata";
 import { meme, MMotif } from "@/components/MotifSearch/MotifUtil";
 import { reverseComplement as rc } from "@/components/tf/geneexpression/utils";
 
+// Add custom colors to Alphabet A and T
```

Comment:
Do we have to set colors manually ? In old FB I am using logojs-react package 2.1.1 and I don't need to set colors manually